react/components/ProductImages/components/Carousel/index.js

```diff
@@ -45,6 +40,8 @@ const initialState = {
   activeIndex: 0,
 }
 
+export const CarouselContext = React.createContext({})
```

Comment:
Put an undefined to the default value of the context. I left an explanation before.